packages/itwinui-css/src/tree/tree.scss

```diff
@@ -41,37 +44,34 @@ $iui-expander-button-width: calc(var(--iui-component-height-small) + 2px);
 .iui-tree-node {
   display: flex;
   cursor: pointer;
-  padding-inline: var(--iui-size-xs);
+  padding-inline: var(--_iui-tree-node-padding-inline);
   align-items: center;
   border-radius: var(--iui-border-radius-1);
 
   &-checkbox {
-    margin-inline-end: var(--iui-size-xs);
+    margin-inline-end: var(--_iui-tree-node-padding-inline);
   }
 
   &-content {
     display: flex;
+    column-gap: var(--_iui-tree-node-padding-inline);
     align-items: center;
-    min-block-size: calc(var(--iui-size-s) * 3);
-    margin-inline-start: calc(#{$iui-expander-button-width} * (var(--level, 0)));
+    min-block-size: var(--_iui-tree-row-height);
+    margin-inline-start: calc(var(--_iui-expander-button-width) * 0.5 * var(--level, 0));
     overflow: hidden;
     padding-inline-start: var(--iui-size-3xs);
     inline-size: 100%;
 
     &-icon {
-      margin-inline: $iui-expander-inline-padding;
+      margin-inline: var(--_iui-tree-node-content-icon-margin-inline);
 
-      &,
       svg {
```

Comment:
this looks unrelated, should be undone. I think it was done this way because the `-content-icon` class could be applied on the `svg` itself

Reply:
In the small tree I am using this:
```sass
<svg-placeholder data-iui-icon-size="s"></svg-placeholder>
```

If I remove the `&,` we get this (pink added to help with the visuals):
![Screenshot 2023-12-07 at 1 07 21 PM](https://github.com/iTwin/iTwinUI/assets/849817/b3e6f9e2-80a4-4895-abde-014e33af2f38)

If I add the `&,` back, we get this alignment issue:
![Screenshot 2023-12-07 at 1 07 45 PM](https://github.com/iTwin/iTwinUI/assets/849817/e502da97-aba4-4fd6-ab45-01e4c5289e08)

Reply:
I guess then we can exclude that specific case using `:not()`?

```suggestion
      &:not([data-iui-icon-size='s']),
      svg {
```

Reply:
Some more context:

In React, the user only passes the actual svg, and we do not use the `Icon` component (which consists of `iui-svg-icon` and `data-iui-icon-size`). In the next major, we should replace all uses of the `iui-icon-style`/`iui-icon-color` mixins with `iui-svg-icon`, but right now we need to avoid such breaking changes.

![](https://github.com/iTwin/iTwinUI/assets/9084735/d4581a74-a3c2-448c-b91c-e7266fab584f)

I think the simplest way would be to include `iui-icon-style(m)` for default tree and `iui-icon-style(s)` for small tree, and keep the html unchanged.

Reply:
Ahh, right, React handles icon differently.  Dang, I'm getting rusty.

`&,` added back in and some CSS added back at the top to set the icon mixin to S for small tree.  Also cleaned up the HTML by removing `.iui-svg-icon` and the size.

Reply:
Change made.